README.txt
                                     README
                                 RasMol 2.7.5.2

  * RasMol Latest Windows                            * RasMol 2.7.5.2 Windows    
    Installer                                          Installer                 
                                                                                 
  * RasMol Latest Source                             * RasMol 2.7.5.2 Source     
    Tarball                          RasMol            Tarball                   
                                                                                 
  * RasMol Latest Manual                             * RasMol 2.7.5 Manual       
                                                                                 
  * Donate to Support RasMol                         * Donate to Support RasMol  
  * Register your RasMol                             * Register your RasMol      

                     Molecular Graphics Visualisation Tool
                         3 June 2009 (rev. 13 May 2011)

                       Based on RasMol 2.6 by Roger Sayle
Biomolecular Structures Group, Glaxo Wellcome Research & Development, Stevenage,
                               Hertfordshire, UK
             Version 2.6, August 1995, Version 2.6.4, December 1998
                      Copyright (c) Roger Sayle 1992-1999

                              and Based on Mods by

+------------------------------------------------------------------------------+
| Author            | Version, Date        | Copyright                         |
|-------------------+----------------------+-----------------------------------|
| Arne Mueller      | RasMol 2.6x1 May     | (c) Arne Mueller 1998             |
|                   | 1998                 |                                   |
|-------------------+----------------------+-----------------------------------|
| Gary Grossman and | RasMol 2.5-ucb       | (c) UC Regents/ModularCHEM        |
| Marco Molinaro    | November 1995        | Consortium 1995, 1996             |
|                   | RasMol 2.6-ucb       |                                   |
|                   | November 1996        |                                   |
|-------------------+----------------------+-----------------------------------|
| Philippe Valadon  | RasTop 1.3 August    | (c) Philippe Valadon 2000         |
|                   | 2000                 |                                   |
|-------------------+----------------------+-----------------------------------|
| Herbert J.        | RasMol 2.7.0 March   | (c) Herbert J. Bernstein          |
| Bernstein         | 1999                 | 1998-2009                         |
|                   | RasMol 2.7.1 June    |                                   |
|                   | 1999                 |                                   |
|                   | RasMol 2.7.1.1       |                                   |
|                   | January 2001         |                                   |
|                   | RasMol 2.7.2 August  |                                   |
|                   | 2000                 |                                   |
|                   | RasMol 2.7.2.1 April |                                   |
|                   | 2001                 |                                   |
|                   | RasMol 2.7.2.1.1     |                                   |
|                   | January 2004         |                                   |
|                   | RasMol 2.7.3         |                                   |
|                   | February 2005        |                                   |
|                   | RasMol 2.7.3.1 Apr   |                                   |
|                   | 06                   |                                   |
|                   | RasMol 2.7.4 Nov 07  |                                   |
|                   | RasMol 2.7.4.1 Jan   |                                   |
|                   | 08                   |                                   |
|                   | RasMol 2.7.4.2 Mar   |                                   |
|                   | 08                   |                                   |
|                   | RasMol 2.7.5 June    |                                   |
|                   | 2009                 |                                   |
|                   | RasMol 2.7.5.1 July  |                                   |
|                   | 2009                 |                                   |
|                   | RasMol 2.7.5.2 May   |                                   |
|                   | 2011                 |                                   |
+------------------------------------------------------------------------------+

   RasMol 2.7.5 incorporates changes by T. Ikonen, G. McQuillan, N. Darakev
   and L. Andrews (via the neartree package). Work on RasMol 2.7.5 supported
   in part by grant 1R15GM078077-01 from the National Institute of General
   Medical Sciences (NIGMS), U.S. National Institutes of Health and by grant
   ER63601-1021466-0009501 from the Office of Biological & Environmental
   Research (BER), Office of Science, U. S. Department of Energy. RasMol
   2.7.4 incorporated changes by G. Todorov, Nan Jia, N. Darakev, P.
   Kamburov, G. McQuillan, and J. Jemilawon. Work on RasMol 2.7.4 supported
   in part by grant 1R15GM078077-01 from the NIGMS/NIH and grant
   ER63601-1021466-0009501 from BER/DOE. RasMol 2.7.3 incorporates changes by
   Clarice Chigbo, Ricky Chachra, and Mamoru Yamanishi. Work on RasMol 2.7.3
   supported in part by grants DBI-0203064, DBI-0315281 and EF-0312612 from
   the U.S. National Science Foundation and grant DE-FG02-03ER63601 from
   BER/DOE. The content is solely the responsibility of the authors and does
   not necessarily represent the official views of the funding organizations.

   The code for use of RasMol under GTK in RasMol 2.7.4.2 was written by
   Teemu Ikonen.
                       and Incorporating Translations by

+-----------------------------------------------------------------------------+
| Author                               | Item                     | Language  |
|--------------------------------------+--------------------------+-----------|
| Isabel Servan Martinez,              | 2.6 Manual               | Spanish   |
| Jose Miguel Fernandez Fernandez      |                          |           |
|--------------------------------------+--------------------------+-----------|
| Jose Miguel Fernandez Fernandez      | 2.7.1 Manual             | Spanish   |
|--------------------------------------+--------------------------+-----------|
| Fernando Gabriel Ranea               | 2.7.1 menus and messages | Spanish   |
|--------------------------------------+--------------------------+-----------|
| Jean-Pierre Demailly                 | 2.7.1 menus and messages | French    |
|--------------------------------------+--------------------------+-----------|
| Giuseppe Martini, Giovanni Paolella, | 2.7.1 menus and messages | Italian   |
| A. Davassi, M. Masullo, C. Liotto    | 2.7.1 help file          |           |
|--------------------------------------+--------------------------+-----------|
| G. Pozhvanov                         | 2.7.3 menus and messages | Russian   |
|--------------------------------------+--------------------------+-----------|
| G. Todorov                           | 2.7.3 menus and messages | Bulgarian |
|--------------------------------------+--------------------------+-----------|
| Nan Jia, G. Todorov                  | 2.7.3 menus and messages | Chinese   |
|--------------------------------------+--------------------------+-----------|
| Mamoru Yamanishi, Katajima Hajime    | 2.7.3 menus and messages | Japanese  |
+-----------------------------------------------------------------------------+

                                This Release by
   Herbert J. Bernstein, Bernstein + Sons, 5 Brewster Lane, Bellport, NY, USA
                          yaya@bernstein-plus-sons.com
                  Copyright (c) Herbert J. Bernstein 1998-2009

   The original RasMol manual was created by Roger Sayle. In July 1996, Dr.
   Margaret Wong of the Chemistry Department, Swinburne University of
   Technology, Australia, made extensive revisions to the RasMol 2.5 manual
   to accurately reflect the operation of RasMol 2.6. Eric Martz of the
   University of Massachusetts made further revisions. In May 1997, William
   McClure of Carnegie Mellon University reorganized the HTML version of the
   manual into multiple sections which could be downloaded quickly and added
   use of frames. Portions of the 2.7.1 version of the RasMol manual were
   derived with permission from William McClure's version using Roger Sayle's
   rasmol.doc for version 2.6.4 as the primary source. Changes were made in
   August 2000 for RasMol version 2.7.2, January 2001 for RasMol version
   2.7.1.1 and April 2001 for RasMol version 2.7.2.1 and February 2005 for
   RasMol version 2.7.3.

                     Documentation Last Updated 14 May 2011
            Edited by Herbert J. Bernstein and Frances C. Bernstein

                                  Translations

   Thanks to the efforts of Jose Miguel Fernandez Fernandez (Departamento de
   Bioquimica y Biologia Molecular. Universidad de Granada. Espana
   (jmfernan@ugr.es)) a translation of the Manual for Rasmol version 2.7.1
   into Spanish is now available. La traduccion espanola del manual de la
   version de la Dra. Wong revisada por Eric Martz fue realizada por Isabel
   Servan Martinez y Jose Miguel Fernandez Fernandez. La actual traduccion
   del Manual de RasMol 2.7.1 ha sido realizada usando como base la anterior
   de RasMol 2.6 por  Jose Miguel Fernandez Fernandez.

   Thanks to translations by Fernando Gabriel Ranea <davinci@dinamica.com.ar>
   in late 2000 and early 2001, RasMol is now capable of rendering most menu
   items and messages in Spanish. Jean-Pierre Demailly
   <Jean-Pierre.Demailly@ujf-grenoble.fr> provided French translations of
   menus and messages in January 2001. Giuseppe Martini
   <martini@iigb.na.cnr.it> and Giovanni Paolella <paolella@dbbm.unina.it>
   with contributions by A. Davassi, M. Masullo and C. Liotto provided
   Italian translations of menus and messages in March 2001.

     ----------------------------------------------------------------------

                      IMPORTANT - Copying and Distribution

   This version is based directly on RasMol version 2.7.5.1, on RasMol
   version 2.7.4.2, on RasMol version 2.7.4.1, on RasMol version 2.7.4, on
   RasMol version 2.7.3.1, on RasMol version 2.7.3, on RasMol version
   2.7.2.1.1, on RasMol version 2.7.2, on RasMol version 2.7.1, on RasMol
   version 2.6_CIF.2, on RasMol version 2.6x1, on RasMol version 2.6.4, and
   RasMol 2.5-ucb and 2.6-ucb.

   Please read the file NOTICE for important notices which apply to this
   package and for license terms (GPL or RASLIC).

     ----------------------------------------------------------------------

  Contents

     * IMPORTANT - Copying and Distribution
     * Installation Instructions
     * Changes
     * Things To Do
     * Introduction
     * Source Code and Binaries
     * Directories
          * ChangeLog -- Full ChangeLog directory
          * src -- source code
               * mac -- Macintosh build directory
               * mswin -- Windows build directory
          * doc -- documentation
          * data -- sample data files

     ----------------------------------------------------------------------

                                  Introduction

   This posting is made to announce the release of RasMol version 2.7.5.2
   This release is based on RasMol 2.7.5.1.

   This is a final update to the 2.7.5 series before release of the 2.7.6
   release candidates, correcting some minor bugs and speeding up surface
   renderings by use of NearTree 3.1. The version of CBFlib used has also
   been updated. The changes made between the 2.7.5.1 release of 23 July 2009
   and the 2.7.5.2 release were:

     * Faster surface rendering by use of NearTree 3.1
     * Updates to GTK version from T. Ikonen
     * Use of CBFlib 0.9.2
     * Use of CQRlib 1.1.2
     * Minor bug fixes

   The changes made between the 2.7.5 release candidate release of 17 July
   2009 and the formal release on 23 July 2009 were:
     * Correction to the support for core CIF data file loads that was
       disabled in the move to CBFlib in place of the internal CIF support.
     * Correction to the CCP4 map read logic in the case of symmetry lines.
       Thanks to Marian Szebenyi for finding this bug.
     * Clarification to the install instructions for 64-bit unix systems.
       Thanks to Marian Szebenyi and Mark Diekhans for pointing out the lack
       of clarity.

   The major changes from the 2.7.4.2 release to the 2.7.5 release include:
     * Support for SBEVSL movie commands.
     * Support for Lee-Richards surface approximation by contouring
       pseudo-Gaussian electron densities.
     * Selection of atoms by proximity to map contours
     * Coloring of maps by the colors of neighboring atoms
     * Significant improvements to the GTK version by Teemu Ikonen
   The 2.7.4.2 release was based on RasMol 2.7.4 and RasMol 2.7.4.1 and
   RasMol 2.7.3.1, the final reference release for the 2.7.3 series. The
   changes from the 2.7.3.1 release to the 2.7.4.1 release included:
     * Support for maps.
     * Message and menu translations for Russian, Bulgarian, Japanese and
       Chinese. Our thanks to G. Pozhvanov, G. Todorov, Nan Jia, Mamoru
       Yamanishi and Katajima Hajime.
     * Fix torsion angle calculation as per bug report and patch by Swati
       Jain.
     * Corrections by Ladislav Michnovic to port to more platforms.
     * Code to read remediated PDB entries as suggested by Huanwang Yang
     * Updated icons.
     * Extended export menus.
   For more detail on these and earlier changes, consult the openrasmol svn
   repository on sourceforge.

   This version is available for MS Windows and various unix-systems. As
   binaries become available, they will be released on
   http://blondie.dowling.edu/projects/rasmol and
   http:/www.sourceforge.net/projects/openrasmol.

   For installation instructions see "INSTALL".

   For a list of open issues in this version, see "TODO".

   RasMol is a molecular graphics program intended for the visualisation of
   proteins, nucleic acids and small molecules. The program is aimed at
   display, teaching and generation of publication quality images. The
   original program was been developed at the University of Edinburgh's
   Biocomputing Research Unit and the Biomolecular Structures Group at Glaxo
   Research and Development, Greenford, UK.

   RasMol reads in molecular co-ordinate files in a number of formats and
   interactively displays the molecule on the screen in a variety of colour
   schemes and representations. Currently supported input file formats
   include Brookhaven Protein Databank (PDB), Tripos' Alchemy and Sybyl Mol2
   formats, Molecular Design Limited's (MDL) Mol file format, Minnesota
   Supercomputer Center's (MSC) XMol XYZ format, CHARMm format, MOPAC format,
   CIF format and mmCIF format files. If connectivity information and/or
   secondary structure information is not contained in the file this is
   calculated automatically. The loaded molecule may be shown as wireframe,
   cylinder (drieding) stick bonds, alpha-carbon trace, spacefilling (CPK)
   spheres, macromolecular ribbons (either smooth shaded solid ribbons or
   parallel strands), hydrogen bonding and dot surface. Atoms may also be
   labelled with arbitrary text strings. Alternate conformers and multiple
   NMR models may be specially coloured and identified in atom labels.
   Different parts of the molecule may be displayed and coloured
   independently of the rest of the molecule or shown in different
   representations simultaneously. The space filling spheres can even be
   shadowed. The displayed molecule may be rotated, translated, zoomed,
   z-clipped (slabbed) interactively using either the mouse, the scroll bars,
   the command line or an attached dials box. RasMol can read a prepared list
   of commands from a `script' file (or via interprocess communication) to
   allow a given image or viewpoint to be restored quickly. RasMol can also
   create a script file containing the commands required to regenerate the
   current image. Finally the rendered image may be written out in a variety
   of formats including both raster and vector PostScript, GIF, PPM, BMP,
   PICT, Sun rasterfile or as a MolScript input script or Kinemage.

   Versions of RasMol have run on a wide range of architectures and systems
   including SGI, sun4, sun3, sun386i, SGI, DEC, HP and E&S workstations, IBM
   RS/6000, Cray, Sequent, DEC Alpha (OSF/1, OpenVMS and Windows NT), IBM PC
   (under Microsoft Windows, Windows NT, OS/2, Linux, BSD386 and *BSD), Apple
   Macintosh (System 7.0 or later), PowerMac and VAX VMS (under DEC Windows).
   UNIX and VMS versions require an 8bit, 24bit or 32bit X Windows frame
   buffer (X11R4 or later). The X Windows version of RasMol provides optional
   support for a hardware dials box and accelerated shared memory rendering
   (via the XInput and MIT-SHM extensions) if available.

   Reports of builds and/or problems on various platforms appreciated.


ChangeLog.txt
                                   ChangeLog
                                 RasMol 2.7.5.2

  * RasMol Latest Windows                            * RasMol 2.7.5.2 Windows    
    Installer                                          Installer                 
                                                                                 
  * RasMol Latest Source                             * RasMol 2.7.5.2 Source     
    Tarball                          RasMol            Tarball                   
                                                                                 
  * RasMol Latest Manual                             * RasMol 2.7.5 Manual       
                                                                                 
  * Donate to Support RasMol                         * Donate to Support RasMol  
  * Register your RasMol                             * Register your RasMol      

                             Molecular Graphics Visualisation Tool
                        13 June 2009 (rev. 13 May 2011)

                              Based on RasMol 2.6 by Roger Sayle
Biomolecular Structures Group, Glaxo Wellcome Research & Development, Stevenage,
                               Hertfordshire, UK
                    Version 2.6, August 1995, Version 2.6.4, December 1998
                              Copyright (c) Roger Sayle 1992-1999

                                     and Based on Mods by

+------------------------------------------------------------------------------+
| Author               | Version, Date                   | Copyright                          |
|-------------------+----------------------+-----------------------------------|
| Arne Mueller      | RasMol 2.6x1 May     | (c) Arne Mueller 1998             |
|                   | 1998                 |                                   |
|-------------------+----------------------+-----------------------------------|
| Gary Grossman and | RasMol 2.5-ucb       | (c) UC Regents/ModularCHEM        |
| Marco Molinaro    | November 1995        | Consortium 1995, 1996             |
|                   | RasMol 2.6-ucb       |                                   |
|                   | November 1996        |                                   |
|-------------------+----------------------+-----------------------------------|
| Philippe Valadon  | RasTop 1.3 August    | (c) Philippe Valadon 2000         |
|                   | 2000                 |                                   |
|-------------------+----------------------+-----------------------------------|
| Herbert J.        | RasMol 2.7.0 March   | (c) Herbert J. Bernstein          |
| Bernstein         | 1999                 | 1998-2011                         |
|                   | RasMol 2.7.1 June    |                                   |
|                   | 1999                 |                                   |
|                   | RasMol 2.7.1.1       |                                   |
|                   | January 2001         |                                   |
|                   | RasMol 2.7.2 August  |                                   |
|                   | 2000                 |                                   |
|                   | RasMol 2.7.2.1 April |                                   |
|                   | 2001                 |                                   |
|                   | RasMol 2.7.2.1.1     |                                   |
|                   | January 2004         |                                   |
|                   | RasMol 2.7.3         |                                   |
|                   | February 2005        |                                   |
|                   | RasMol 2.7.3.1 Apr   |                                   |
|                   | 06                   |                                   |
|                      | RasMol 2.7.4 Nov 07             |                                    |
|                   | RasMol 2.7.4.1 Jan   |                                   |
|                   | 08                   |                                   |
|                   | RasMol 2.7.4.2 Mar   |                                   |
|                   | 08                   |                                   |
|                   | RasMol 2.7.5 Jun 09  |                                   |
|                   | RasMol 2.7.5.1 Jul   |                                   |
|                   | 09                   |                                   |
|                   | RasMol 2.7.5.2 May   |                                   |
|                   | 11                   |                                   |
+------------------------------------------------------------------------------+

                                        This Release by
          Herbert J. Bernstein, Bernstein + Sons, 5 Brewster Lane, Bellport, NY, USA
                                 yaya@bernstein-plus-sons.com
                         Copyright (c) Herbert J. Bernstein 1998-2009

     ----------------------------------------------------------------------

                                   IMPORTANT

   This version is based directly on RasMol 2.7.5.1, RasMol 2.7.4.2, RasMol
   2.7.4.1, RasMol 2.7.4, RasMol 2.7.3.1, RasMol 2.7.3, RasMol 2.7.2.1.1 and
   on RasMol 2.7.2, on RasMol 2.7.1.1, on RasTop 1.3, on RasMol version
   2.6_CIF.2, on RasMol version 2.6x1, on RasMol version 2.6.4, and RasMol
   2.5-ucb and 2.6-ucb. Please read the file NOTICE for important notices
   which apply to this package and for license terms (GPL or RASLIC).

   This version of RasMol is not in the public domain, but it is given freely
   to the community in the hopes of advancing science. If you make changes,
   please make them in a responsible manner, and please offer us the
   opportunity to include those changes in future versions of RasMol.

     ----------------------------------------------------------------------


src/font.h
/***************************************************************************
 *                              RasMol 2.7.5                               *
 *                                                                         *
 *                                 RasMol                                  *
 *                 Molecular Graphics Visualisation Tool                   *
 *                              13 June 2009                               *
 *                                                                         *
 *                   Based on RasMol 2.6 by Roger Sayle                    *
 * Biomolecular Structures Group, Glaxo Wellcome Research & Development,   *
 *                      Stevenage, Hertfordshire, UK                       *
 *         Version 2.6, August 1995, Version 2.6.4, December 1998          *
 *                   Copyright (C) Roger Sayle 1992-1999                   *
 *                                                                         *
 *                          and Based on Mods by                           *
 *Author             Version, Date             Copyright                   *
 *Arne Mueller       RasMol 2.6x1   May 98     (C) Arne Mueller 1998       *
 *Gary Grossman and  RasMol 2.5-ucb Nov 95     (C) UC Regents/ModularCHEM  *
 *Marco Molinaro     RasMol 2.6-ucb Nov 96         Consortium 1995, 1996   *
 *                                                                         *
 *Philippe Valadon   RasTop 1.3     Aug 00     (C) Philippe Valadon 2000   *
 *                                                                         *
 *Herbert J.         RasMol 2.7.0   Mar 99     (C) Herbert J. Bernstein    * 
 *Bernstein          RasMol 2.7.1   Jun 99         1998-2008               *
 *                   RasMol 2.7.1.1 Jan 01                                 *
 *                   RasMol 2.7.2   Aug 00                                 *
 *                   RasMol 2.7.2.1 Apr 01                                 *
 *                   RasMol 2.7.2.1.1 Jan 04                               *
 *                   RasMol 2.7.3   Feb 05                                 *
 *                   RasMol 2.7.3.1 Apr 06                                 *
 *                   RasMol 2.7.4   Nov 07                                 *
 *                   RasMol 2.7.4.1 Jan 08                                 *
 *                   RasMol 2.7.4.2 Mar 08                                 *
 *                   RasMol 2.7.5   May 09                                 *
 *                                                                         *
 * RasMol 2.7.5 incorporates changes by T. Ikonen, G. McQuillan, N. Darakev*
 * and L. Andrews (via the neartree package).  Work on RasMol 2.7.5        *
 * supported in part by grant 1R15GM078077-01 from the National Institute  *
 * of General Medical Sciences (NIGMS), U.S. National Institutes of Health *
 * and by grant ER63601-1021466-0009501 from the Office of Biological &    *
 * Environmental Research (BER), Office of Science, U. S. Department of    *
 * Energy.  RasMol 2.7.4 incorporated  changes by G. Todorov, Nan Jia,     *
 * N. Darakev, P. Kamburov, G. McQuillan, and J. Jemilawon. Work on RasMol *
 * 2.7.4 supported in part by grant 1R15GM078077-01 from the NIGMS/NIH and *
 * grant ER63601-1021466-0009501 from BER/DOE.  RasMol 2.7.3 incorporates  *
 * changes by Clarice Chigbo, Ricky Chachra, and Mamoru Yamanishi.  Work   *
 * on RasMol 2.7.3 supported in part by grants DBI-0203064, DBI-0315281    *
 * and EF-0312612 from the U.S. National Science Foundation and grant      *
 * DE-FG02-03ER63601 from BER/DOE. The content is solely the responsibility*
 * of the authors and does not necessarily represent the official views of *
 * the funding organizations.                                              *
 *                                                                         *
 * The code for use of RasMol under GTK in RasMol 2.7.4.2 and 2.7.5 was    *
 * written by Teemu Ikonen.                                                *
 *                                                                         *
 *                    and Incorporating Translations by                    *
 *  Author                               Item                     Language *
 *  Isabel Servan Martinez,                                                *
 *  Jose Miguel Fernandez Fernandez      2.6   Manual             Spanish  *
 *  Jose Miguel Fernandez Fernandez      2.7.1 Manual             Spanish  *
 *  Fernando Gabriel Ranea               2.7.1 menus and messages Spanish  *
 *  Jean-Pierre Demailly                 2.7.1 menus and messages French   *
 *  Giuseppe Martini, Giovanni Paolella, 2.7.1 menus and messages          *
 *  A. Davassi, M. Masullo, C. Liotto    2.7.1 help file          Italian  *
 *  G. Pozhvanov                         2.7.3 menus and messages Russian  *
 *  G. Todorov                           2.7.3 menus and messages Bulgarian*
 *  Nan Jia, G. Todorov                  2.7.3 menus and messages Chinese  *
 *  Mamoru Yamanishi, Katajima Hajime    2.7.3 menus and messages Japanese *
 *                                                                         *
 *                             This Release by                             *
 * Herbert J. Bernstein, Bernstein + Sons, 5 Brewster Ln, Bellport, NY, USA*
 *                       yaya@bernstein-plus-sons.com                      *
 *               Copyright(C) Herbert J. Bernstein 1998-2008               *
 *                                                                         *
 *                READ THE FILE NOTICE FOR RASMOL LICENSES                 *
 *Please read the file NOTICE for important notices which apply to this    *
 *package and for license terms (GPL or RASLIC).                           *
 ***************************************************************************/
/* font.h
 $Log$
 Revision 1.5  2008/03/22 18:42:51  yaya
 Post release cleanup and credit to Ikonen in file headers. -- HJB

 Revision 1.4  2008/03/17 03:01:31  yaya
 Update to agree with 2.7.4.2 release and T. Ikonen GTK mods -- HJB

 Revision 1.3  2008/03/17 01:32:41  yaya
 Add gtk mods by tpikonen, and intergate with 2.7.4.2 mods -- HJB

 Revision 1.3  2008/03/16 22:25:21  yaya
 Align comments with production version; Update rasmol_install and
 rasmol_run shell scripts for Japanese and Chinese; Align logic for
 positioning and sizing initial window with windows version -- HJB

 Revision 1.2  2008/01/28 03:29:37  yaya
 Update CVS to RasMol_2.7.4.1 -- HJB

 Revision 1.2  2007/11/19 03:28:39  yaya
 Update to credits for 2.7.4 in manual and headers
 Mask code added -- HJB

 Revision 1.1.1.1  2007/03/01 01:16:33  todorovg
 Chinese working versio from rasmol_ru initial import

 Revision 1.4  2007/02/02 21:08:59  yaya
 Remove // comments for AIX build
 Fix signed length conflict
 Fix array bounds violation for molecule names in multiple.c -- HJB

 Revision 1.3  2006/11/01 03:23:50  yaya
 Update NSIS windows installer for more script types and to fix
 misplaced script instructions for data files; add document and
 script icons directly in raswin.exe; add credit line to
 G. A. Pozhvanov in comments for Russian translations. -- HJB

 Revision 1.2  2006/09/17 10:53:55  yaya
 Clean up headers and start on code for X11 -- HJB

 Revision 1.1.1.1  2006/09/16 18:45:45  yaya
 Start of RasMol Russian Translation Project based on translations
 by Gregory A. Pozhvanov of Saint Petersburg State University -- HJB

 Revision 1.1.1.1  2006/06/19 22:05:14  todorovg
 Initial Rasmol 2.7.3 Import

 Revision 1.1  2004/05/07 19:46:16  yaya
 Initial revision

 Revision 1.2  2004/02/15 00:24:00  yaya
 *** empty log message ***

 Revision 1.1  2003/12/12 21:10:38  yaya
 Initial revision

 Revision 1.1  2001/01/31 02:13:45  yaya
 Initial revision

 Revision 1.3  2000/08/26 18:12:55  yaya
 Updates to header comments in all files

 Revision 1.2  2000/08/09 01:18:32  yaya
 Rough cut with ucb

 */
/*font declarations for pixutils.c and outfile.c*/

#ifdef PIXUTILS
	char FontDimen[23];
	int FontWid[97];

   	char *VectFont[97] = {
    /*  32 ' ' */  "",
    /*  33 '!' */  "ChdhdicichCkdkewbwck",
    /*  34 '"' */  "CvbvbwcwcubtFvevewfwfuet",
    /*  35 '#' */  "AlmlAqmqDidtJijt",
    /*  36 '$' */  "AjcihijjkljnhodobparbtduiuktFgfw",
    /*  37 '%' */  "AgmwIhlhmimllmimhlhiihCqfqgrgufvcvbubrcq",
    /*  38 '&' */  "Kgihbsbtcvewfwgugsfqboamakbichegfghhkk",
    /*  39 ''' */  "Evdvdweweucs",
    /*  40 '(' */  "Egchbiakasbucvew",
    /*  41 ')' */  "Agchdiekesducvaw",
    /*  42 '*' */  "AplpCkjuCujk",
    /*  43 '+' */  "CokoGkgs",
    /*  44 ',' */  "Dgcgchdhdgdfce",
    /*  45 '-' */  "Coko",
    /*  46 '.' */  "Ggfgfhghgg",
    /*  47 '/' */  "Agmw",
    /*  48 '0' */  "EwcvbtbjchegigkhljltkviwewChkv",
    /*  49 '1' */  "DtgwggDgjg",
    /*  50 '2' */  "Kgagaibkhojpkrktjvhwdwbvat",
    /*  51 '3' */  "AtbvdwhwjvktkrjphoeoHojnklkjjhhgdgbhaj",
    /*  52 '4' */  "Klalhwhg",
    /*  53 '5' */  "Kwawaocphpjokmkjjhhgcgah",
    /*  54 '6' */  "Jvhwdwbvatajbhdghgjhkjkmjohpdpboan",
    /*  55 '7' */  "Awkwdg",
    /*  56 '8' */  "DwbvatarbpdohojpkrktjvhwdwDobnalajbhdghgjhkjkljnho",
    /*  57 '9' */  "Bhdghgjhkjktjvhwdwbvataqbodnhnjokq",
    /*  58 ':' */  "GififjgjgiGpfpfqgqgp",
    /*  59 ';' */  "DicicjdjdidhcgCpdpdqcqcp",
    /*  60 '<' */  "Lgbolw",
    /*  61 '=' */  "AsisAnjn",
    /*  62 '>' */  "Bglobw",
    /*  63 '?' */  "ArasbucvewgwivjuktkrjphognflfjFhfgggghfh",
    /*  64 '@' */  "Knjlhkflenepfrhsjrkqkkljmjnkomosnumvkwewcvbuasakbichegkg",
    /*  65 'A' */  "AgcmhwmmogCmmm",
    /*  66 'B' */  "AgigkhlimklmkniokplqmslukviwawBwbgboio",
    /*  67 'C' */  "Milhjgegchbiakasbucvewjwlvmu",
    /*  68 'D' */  "AgigkhlimkmslukviwawBwbg",
    /*  69 'E' */  "KwawagkgAoho",
    /*  70 'F' */  "KwawagAoho",
    /*  71 'G' */  "MilhjgegchbiakasbucvewjwlvmuMgmmHmom",
    /*  72 'H' */  "AgawKgkwAoko",
    /*  73 'I' */  "AgggAwgwDgdw",
    /*  74 'J' */  "AmakbichegghhiikiwDwnw",
    /*  75 'K' */  "AgawAmkwCokg",
    /*  76 'L' */  "Kgagaw",
    /*  77 'M' */  "Agawhmowog",
    /*  78 'N' */  "Agawmgmw",
    /*  79 'O' */  "Akbichegigkhlimkmslukviwewcvbuasak",
    /*  80 'P' */  "Agawiwkvlumsmrlpkoinan",
    /*  81 'Q' */  "AkbichegigkhlimkmslukviwewcvbuasakHlmg",
    /*  82 'R' */  "AgawiwkvlumsmrlpkoinanInmg",
    /*  83 'S' */  "Ajchegigkhlimklmkncpbqasbucvewiwkvmt",
    /*  84 'T' */  "HghwAwow",
    /*  85 'U' */  "Awakbichegigkhlimkmw",
    /*  86 'V' */  "Awggmw",
    /*  87 'W' */  "Aweghokgow",
    /*  88 'X' */  "AwkgAgkw",
    /*  89 'Y' */  "AwfokwFgfo",
    /*  90 'Z' */  "Awkwagkg",
    /*  91 '[' */  "Igdgdwiw",
    /*  92 '\' */  "Awmg",
    /*  93 ']' */  "Dgigiwdw",
    /*  94 '^' */  "Cqgwkq",
    /*  95 '_' */  "Bglg",
    /*  96 '`' */  "Cvdvdwcwcues",
    /*  97 'a' */  "BqdrirjqkokhlgKhigdgbhajbldmimkk",
    /*  98 'b' */  "BichegggihjikkkmjoipgqeqcpboAwbwbgag",
    /*  99 'c' */  "Jqhrercqbpanakbicheghgjh",
    /* 100 'd' */  "KiggegchbiakanbpcqerhrjqkoLgkgkwlw",
    /* 101 'e' */  "Kiihggegchbiakanbpcqerhrjqkokmam",
    /* 102 'f' */  "BgdgCgcrdtfuhtirAnfn",
    /* 103 'g' */  "KiihggegchbiakambodphpjokmLpkpkdjbhaeacbbc",
    /* 104 'h' */  "AwbwbgagBncpeqhqjpknkg",
    /* 105 'i' */  "AgegCgcpbpCrcsbsbrcr",
    /* 106 'j' */  "GshshrgrgsGphphegcebdbbcae",
    /* 107 'k' */  "AgbgbwawBjhpIgem",
    /* 108 'l' */  "AgegCgcubu",
    /* 109 'm' */  "AqbqbgBncpeqfqhpiniginjplqmqoppnpg",
    /* 110 'n' */  "AqbqbgagBncpeqhqjpknkg",
    /* 111 'o' */  "Akbichegggihjikkkmjoipgqeqcpboamak",
    /* 112 'p' */  "BichegggihjikkkmjoipgqeqcpboAqbqbaaa",
    /* 113 'q' */  "KiggegchbiakanbpcqerhrjqkoLpkpkdlbna",
    /* 114 'r' */  "AqbqbgBncpeqhqjo",
    /* 115 's' */  "Ahcgggihjjikbmanbpdqhqjp",
    /* 116 't' */  "AqiqEuehfghgih",
    /* 117 'u' */  "AqakbichegggihjikkKqkglg",
    /* 118 'v' */  "Arfgkr",
    /* 119 'w' */  "Areghmkgor",
    /* 120 'x' */  "ArkgAgkr",
    /* 121 'y' */  "AqakbichegggihjikkKqkdjbhaeacbbc",
    /* 122 'z' */  "Arkragkg",
    /* 123 '{' */  "Egchbjblcnaocpbrbtcvew",
    /* 124 '|' */  "Gggw",
    /* 125 '}' */  "Agchdjdlcneocpdrdtcvaw",
    /* 126 '~' */  "Arbtcueuftfsgrirjsku",
    /* 127 'Angstroms' */  "AgdkhqlkogDklkHwjuhrfuhw", 
    /* 128 'Degrees' */  "Aqbocnemgminjokqksjuivgwewcvbuasaq" 

        };

#else
	extern char FontDimen[23];
	extern int FontWid[97];
   	extern char *VectFont[97];
#endif
